_includes/modern-appendix.md

```diff
@@ -1220,7 +1220,7 @@ The specified casemappings are as follows:
 * **`rfc1459-strict`**: Same casemapping as `'ascii'`, with the characters `'{'`, `'}'`, and `'|'` being the lower-case equivalents of `'['`, `']'`, and `'\'`, respectively. Note that the difference between this and `rfc1459` above is that in rfc1459-strict, `'^'` and `'~'` are not casefolded.
 * **`rfc7613`**: Proposed casemapping which defines a method based on PRECIS, allowing additional Unicode characters to be correctly casemapped <sup><a href="https://github.com/ircv3/ircv3-specifications/pull/272">[link]</a></sup>.
 
-The value MUST be specified and is a string. Servers MAY advertise alternate casemappings to those above, but clients MAY NOT be able to understand or perform them.
+The value MUST be specified and is a string. Servers MAY advertise alternate casemappings to those above, but clients MAY NOT be able to understand or perform them. If the parameter is not published by the server at all, clients SHOULD assume `CASEMAPPING=rfc1459`.
```

Comment:
Not strictly corresponding to the RFC, but see https://datatracker.ietf.org/doc/html/draft-brocklesby-irc-isupport-03#section-4.5